### PR TITLE
Export windows functions directly from hermod

### PR DESCRIPTION
**Pull request by richfitz**
This package prevents users needing to interact at all with hermod.windows - hermod becomes the wrapper for the call

### CODE REVIEW

File: R/windows.R [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+##'   you will need to use twice as many as you expect (so *four*
+##'   backslashes at the beginning and then two for each separator).
+##'   If this makes you feel bad know that you are not alone:
+##'   https://xkcd.com/1638 -- alternatively you may use forward
+##'   slashes in place of backslashes (e.g. `//fi--didef3.dide.ic.ac.uk/tmp`)
+##'
+##' @param drive_remote The place to mount the drive on the cluster.
+##'   We're probably going to mount things at Q: and T: already so
+##'   don't use those.  And things like C: are likely to be used.
+##'   Perhaps there are some guidelines for this somewhere?
```
Comment (weshinsley) [inline R/windows.R:47]: Future to-dos - 
* Should we have a ticket to write this guidance and improve that last sentence!
* Should we outlaw `I:` here?
* I slightly feel we should drift away from the temp drive, but perhaps not quite ready to shout that yet.

File: drivers/windows/R/zzz.R [hunk 2]
```diff
@@ -1 +1,11 @@
...
+
+
+.onAttach <- function(...) {
+  # nocov start
+  msg <- paste("You have called 'library(hermod.windows)' but you don't need",
+               "to; this package just needs to be installed to work, and you",
+               "only need to call 'library(hermod)'")
+  packageStartupMessage(paste(strwrap(msg), collapse = "\n"))
+  # nocov end
+}
```
Comment (weshinsley) [inline drivers/windows/R/zzz.R:11]: Nice!

**Review by weshinsley — APPROVED** [review]
Looks good - one drive-by edit on the docs

File: R/windows.R (hunk 1 above)
Comment (richfitz) [reply R/windows.R:47]: * https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-4790 - added a ticket for `I:`
* https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-4791 - for docs

For the temp drive we can probably just softly move away from it?

**Pull request merged**